include/ck/utility/data_type.hpp

```diff
@@ -942,6 +942,11 @@ using int8x16_t = typename vector_type<int8_t, 16>::type;
 using int8x32_t = typename vector_type<int8_t, 32>::type;
 using int8x64_t = typename vector_type<int8_t, 64>::type;
 
+#ifdef CK_EXPERIMENTAL_BIT_INT_EXTENSION_INT4
+// i4
+using int4x16_t = typename vector_type<int4_t, 16>::type;
```

Comment:
Did you tested int4 data type? At the time of adding `int4` data type the compiler threw errors in lines: https://github.com/ROCmSoftwarePlatform/composable_kernel/blob/develop/include/ck/utility/data_type.hpp#L369-L372 since extended vector types did not support arbitrary bit-size integer types (`_BitInt`)

Reply:
Thanks for the information, I'll add unit test to it : )

Reply:
Yes, I failed in compile the `int4 `related code. I use the docker 
`compute-artifactory.amd.com:5000/rocm-plus-docker/compute-rocm-dkms-no-npi-hipclang:10738-ubuntu-20.04-stg2`